src/pages/workspace/categories/CategoryForm.tsx

```diff
@@ -28,9 +28,12 @@ type CategoryFormProps = {
 
     /** Function to validate the edited values of the form */
     validateEdit?: (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>) => FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>;
+
+    /** Should go back after submitting the form */
+    isShouldGoBackOnSubmit?: boolean;
```

Comment:
Prefixing `should` is enough to understand it's a boolean.

```suggestion
    shouldGoBackOnSubmit?: boolean;
```